app/assets/javascripts/discourse/lib/dialog-holder/addon/components/dialog-holder.hbs

```diff
@@ -0,0 +1,35 @@
+<div id="dialog-holder" class="dialog-container {{this.dialog.classes}}" aria-labelledby={{this.dialog.titleElementId}} aria-hidden="true">
+  <div class="dialog-overlay" data-a11y-dialog-hide></div>
+
+  {{#if this.dialog.type}}
+    <div class="dialog-content" role="document">
+      {{#if this.dialog.title}}
+        <div class="dialog-header">
+          <h3 id="dialog-title">{{this.dialog.title}}</h3>
+          <DButton @icon="times" @action={{action this.dialog.cancel}} @class="btn-flat dialog-close close" @title="modal.close" />
+        </div>
+      {{/if}}
+
+      {{#if this.dialog.message}}
+        <div class="dialog-body">
+          {{html-safe this.dialog.message}}
```

Comment:
David recommended we avoid any html-safe directly in hbs and prefer to create a safe variable in js we woudl use in the template, this way we avoid having to worry of where `dialog.message`  is used if we change it

Reply:
Good catch, I have disabled this for now because in the current state, no messages have HTML. There are only a few places where we pass in HTML, and in fact, it might not be for the message, but rather for button elements, so no need to handle HTML here now.